bundles.json

```diff
@@ -29,5 +30,7 @@
 
     "example/imagery",
     "example/eventGenerator",
-    "example/generator"
+    "example/generator",
+
+    "testing/dialogTest"
```

Comment:
I'd move this bundle to `example/notification` and remove it from `bundles.json` so that it isn't active by default